ChangeLog

```diff
@@ -31,7 +31,8 @@
 === Miscellaneous ===
 
 === Bug fixes ===
-
+* #1947 (The constructor of FunctionalChaosResult is undocumented)
+* #1444 (The calibration examples are unclear)
```

Comment:
I propose we stop listing bugs here while in-development to avoid the conflicts we get often,
instead we make sure that the bug is added to the github milestone and we'll add them here before the release

Reply:
Good idea!

Reply:
Ok. Meanwhile, I updated it as a last update to fulfill the PR.